build

```diff
@@ -1,23 +0,0 @@
-#!/usr/bin/env python3
-
-import os
-import subprocess
-import sys
-import traceback
-
-from scripts_build import stack_build
-
-
-def main ():
```

Comment:
this has to stay because it is used by manager to build package

Reply:
Ah okay! I was told that it was unused. Will fix it in the next push.